python/composio/cli/add.py

```diff
@@ -58,8 +58,9 @@ class AddIntegrationExamples(HelpfulCmd):
 @click.option(
     "-a",
     "--auth-mode",
-    type=str,
+    type=str.upper,
     help="Specify auth mode for given app.",
```

Comment:
Consider updating the help text to explicitly mention that auth modes are automatically converted to uppercase. This would improve user experience by making the behavior more transparent.